src/index.html

```diff
@@ -1,13 +1,54 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="product">
+      <div class="product__card" data-qa="card">
+        <img
+          src="./images/imac.jpeg"
+          class="product__img"
+          alt="imac"
+        >
+        <span class="product__card-name">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </span>
+        <span class="product__card-code">
+          Код товара: 195434
+        </span>
+        <div class="product__rating">
+          <div class="stars">
+            <div class="star"></div>
```

Comment:
fix each stars' class

```suggestion
          <div class="stars">
            <div class="stars__star"></div>
```